packages/api-tests/test/window.spec.js

```diff
@@ -908,6 +904,24 @@ describe('Window API', function(done) {
         return chainPromises(steps);
       });
     }
+
+    if (process.env.MOCHA_CONTAINER === 'browser') {
```

Comment:
Nitpick: see comment above on `MOCHA_CONTAINER` vs `@no-browser` - should we be more consistent?